### PR TITLE
Allow using internal interfaces (e.g. "Traversable") as parameter types

### LINKED ISSUE #20: The "\Go\ParserReflection\ReflectionParameter::getClass" fails when parameter has internal interface type
The `\Go\ParserReflection\ReflectionParameter::getClass` method works fine with user-defined and internal classes. It uses own `\Go\ParserReflection\ReflectionClass` class) instead of `\ReflectionClass` when method parameter type hint is an internal interface, like `Traversable`.

### PR DESCRIPTION
**Pull request by aik099**
Closes #20 

P.S.
If tests are needed, then I'm happy to provide them if you'd show me some examples. I'm new to library and therefore need some assistance please.

### CODE REVIEW

File: src/ReflectionParameter.php [hunk 1]
```diff
@@ -184,7 +184,7 @@ public function getClass()
                 throw new ReflectionException("Can not resolve a class name for parameter");
             }
             $className   = $parameterType->toString();
-            $classExists = class_exists($className, false);
+            $classExists = class_exists($className, false) || interface_exists($className, false);
```
Comment (lisachenko) [inline src/ReflectionParameter.php]: Parameter should be renamed to $classOrInterfaceExists here

Comment (aik099) [inline src/ReflectionParameter.php]: Done.

**Comment by lisachenko** [comment]
It would be nice to have a test for that case.
Here is a guide, how to do this:
1) Open `FileWithParameters` file and add into `miscParameters` function one more argument with typehint (better to the end of args):
```php
    \Traversable $traversable
```
2) Add a test for this into the `ReflectionParameterTest` into the `testGetClassMethod()` test case:
```php
        $traversableInterface = $parameters[12 /* \Traversable $traversable */]->getClass();
        $this->assertInstanceOf(\ReflectionClass::class, $traversableInterface);
        $this->assertSame(\Traversable::class, $traversableInterface->getName());
```

That's it, now you will have a test case for that

**Comment by aik099** [comment]
Added tests and squashed.

File: tests/Stub/FileWithParameters.php [hunk 2]
```diff
@@ -20,7 +20,8 @@ function miscParameters(
     &$byReferenceParam,
     &$byReferenceNullable = __CLASS__,
     $constParam = TEST_PARAMETER,
-    $constValueParam = __NAMESPACE__ // This line is long and should be truncated
+    $constValueParam = __NAMESPACE__, // This line is long and should be truncated
```
Comment (aik099) [inline tests/Stub/FileWithParameters.php]: Not sure why CS reports an issue here.

**Comment by lisachenko** [comment]
Thanks! Merged

**Pull request merged**